client/context/context.go

```diff
@@ -55,24 +50,24 @@ type CLIContext struct {
 
 // NewCLIContext returns a new initialized CLIContext with parameters from the
 // command line using Viper.
-func NewCLIContext() CLIContext {
+func NewCLIContext(cdc *codec.Codec) *CLIContext {
```

Comment:
All instances of calling `NewCLIContext` also included a `WithCodec` call. So I've collapsed the two functions.